src/wp-includes/class-wp-theme.php

```diff
@@ -293,13 +303,15 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 			$this->headers['Name'] = $this->stylesheet;
 			$this->errors          = new WP_Error( 'theme_stylesheet_not_readable', __( 'Stylesheet is not readable.' ) );
 			$this->template        = $this->stylesheet;
+			$this->is_block_theme();
```

Comment:
See above, this is an invalid theme, so it doesn't matter if it's a block theme or not.
```suggestion
			$this->block_theme = false;
```